gtars/src/common/models/fragments.rs

```diff
@@ -47,6 +47,7 @@ impl From<Fragment> for Region {
             chr: val.chr,
             start: val.start,
             end: val.end,
+            rest: String::new(),
```

Comment:
So for saying so often, but, are we sure `Option<String>` might be a better solution here